pkg/provider/azure.go

```diff
@@ -686,9 +686,8 @@ func (az *Cloud) Instances() (cloudprovider.Instances, bool) {
 }
 
 // InstancesV2 returns an instancesV2 interface. Also returns true if the interface is supported, false otherwise.
-// TODO: implement ONLY for external cloud provider
 func (az *Cloud) InstancesV2() (cloudprovider.InstancesV2, bool) {
-	return nil, false
+	return az, false
```

Comment:
should it be `az, true` indicates it supports InstancesV2? and when are we switching to use InstancesV2, is there e2e test to cover this?

Reply:
Yeah, that would be `true` here. The `InstancesV2` would be enabled once this is merged.

Reply:
Tested manually to make sure the information on the node is good.